gdal/scripts/genfrmts.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
will this file be needed after your work ?

Reply:
it will be removed